backend/backends/__init__.py

```
@@ -708,13 +762,53 @@ def get_tts_backend_for_engine(engine: str) -> TTSBackend:
 
+def drop_tts_backend_for_engine(engine: str) -> bool:
+    """Unload and forget one TTS backend instance so Python can release VRAM refs."""
+    global _tts_backends
+    with _tts_backends_lock:
+        backend = _tts_backends.pop(engine, None)
+    if backend is None:
+        return False
+    try:
+        backend.unload_model()
+    finally:
+        return True
```

Ruff (0.15.12)

[warning] `return` inside `finally` blocks cause exceptions to be silenced

(B012)